com.remarkable.reMarkable.json

```diff
@@ -0,0 +1,71 @@
+{
+    "app-id": "com.remarkable.reMarkable",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.9",
+    "sdk": "org.kde.Sdk",
+    "command": "reMarkable",
+    "finish-args": [
+        "--filesystem=home",
+        "--socket=x11",
+        "--share=ipc",
+        "--share=network",
+        "--extra-data=remarkable-linux-client.tgz:1a1519adc6d62359f6f73bc2a4a6e76fb710fed78ca1bc09e46191e825d3058c:20777403::https://remarkable.engineering/remarkable-linux-client-0.0.5-16-1408-g7eca2b66.tgz",
+        "--extra-data=com.remarkable.reMarkable.png:0754c6fac1e36ca08207a36272af2bfbbc37cbd2e0b93a6447288e8106cf4c64:3437::https://avatars3.githubusercontent.com/u/19650282?s=200&v=4"
+            
+    ],
+    "modules": [
+        {
+            "name": "reMarkable",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -D apply_extra /app/bin/apply_extra",
+                "install -D reMarkable /app/bin/reMarkable"
+            ],
+            "sources": [
+                {
+                    "type": "script",
+                    "dest-filename": "apply_extra",
+                        "commands": [
+                           "tar zxf remarkable-linux-client.tgz --strip-components=1",
+                           "rm remarkable-linux-client.tgz",
+                           "rm README",
+                           "mkdir -p export/share/applications",
+                           "echo '[Desktop Entry]\nVersion=1.0\nType=Application\nName=reMarkable\nIcon=com.remarkable.reMarkable\nExec=\"reMarkable\" %f\nComment=Manage reMarkable device\nTerminal=false' > export/share/applications/com.remarkable.reMarkable.desktop",
+                           "mkdir -p export/share/icons/hicolor/256x256/apps",
+                           "mv com.remarkable.reMarkable.png export/share/icons/hicolor/256x256/apps/"
+                        ]
+                },
+                {
+                    "type": "script",
+                    "dest-filename": "reMarkable",
+                    "commands": [
+                        "exec env LD_LIBRARY_PATH=/app/usr/lib TMPDIR=$XDG_CACHE_HOME /app/extra/reMarkable \"$@\""
+                    ]
+                }
+            ],
+            "modules": [
+                {
+                    "name": "qtwebsockets",
+                    "buildsystem": "simple",
```

Comment:
there's a qmake buildsystem available

Reply:
@nedrichards  Yeah I know, but how do you combine it with the modifications I do with sed?

Reply:
Good point, you can't do those as a post-install hook since they need to run before make